src/Magnum/Platform/GlfwApplication.cpp

```diff
@@ -193,6 +192,24 @@ void GlfwApplication::staticErrorCallback(int, const char* description) {
     Error() << description;
 }
 
+GlfwApplication::InputEvent::Modifiers GlfwApplication::KeyEvent::getCurrentGlfwModifiers(GLFWwindow* window) {
+    static_assert(GLFW_PRESS == true && GLFW_RELEASE == false, "GLFW press and release constants do not correspond to bool values.");
+
+    Modifiers mods = (glfwGetKey(window, Int(Key::LeftShift)) || glfwGetKey(window, Int(Key::RightShift)))
+                     ? Modifiers{Modifier::Shift} : Modifiers{};
+    if(glfwGetKey(window, Int(Key::LeftAlt)) || glfwGetKey(window, Int(Key::RightAlt))) {
+        mods |= Modifier::Alt;
+    }
+    if(glfwGetKey(window, Int(Key::LeftCtrl)) || glfwGetKey(window, Int(Key::RightCtrl))) {
+        mods |= Modifier::Ctrl;
+    }
+    if(glfwGetKey(window, Int(Key::RightSuper))) {
+        mods |= Modifier::AltGr;
+    }
+
+    return mods;
+}
+
```

Comment:
@mosra I tried to clean this up a bit to make it look as least bad as possible. I think this should be okay.